internal/api/hooks.go

```diff
@@ -55,20 +74,176 @@ func (a *API) runPostgresHook(ctx context.Context, tx *storage.Connection, name
 	return response, nil
 }
 
-// invokeHook invokes the hook code. tx can be nil, in which case a new
+func readBodyWithLimit(rsp *http.Response) ([]byte, error) {
+	defer rsp.Body.Close()
+
+	const limit = 20 * 1024 // 20KB
```

Comment:
Move this to a top constant I guess. And maybe give it something like 1-5MB. 20KB is kinda low.

Reply:
This is  the [recommended payload size though](https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md#payload-size)

Perhaps we can move it to 200kb and then increase further if there are complaints? Let me know what you think